docs/word/dictionary-task-pane-add-ins.md

```diff
@@ -84,12 +94,12 @@ The following code shows the XSD for the OfficeDefinitions XML Schema.
 </xs:schema>
 ```
 
-Returned XML that conforms to the OfficeDefinitions schema consists of a root `Result` element that contains a `Definitions` element with from zero to three `Definition` child elements, each of which contains definitions that are no more than 400 characters in length. Additionally, the URL to the full page on the dictionary site must be provided in the `SeeMoreURL` element. The following example shows the structure of returned XML that conforms to the OfficeDefinitions schema.
+Returned XML that conforms to the OfficeDefinitions schema consists of a root **\<Result\>** element that contains a **\<Definitions\>** element with from zero to three **\<Definition\>** child elements, each of which contains definitions that are no more than 400 characters in length. Additionally, the URL to the full page on the dictionary site must be provided in the **\<SeeMoreURL\>** element. The following example shows the structure of returned XML that conforms to the OfficeDefinitions schema.
```

Comment:
```suggestion
Returned XML consists of a root **\<Result\>** element that contains a **\<Definitions\>** element with zero to three **\<Definition\>** child elements. Each child element contains definitions that are less than 400 characters in length. Additionally, the URL to the full page on the dictionary site must be provided in the **\<SeeMoreURL\>** element. The following example shows the structure of returned XML that conforms to the OfficeDefinitions schema.
```
Shortened sentences for readability and localization. Also removed the redundant phrase that's emphasized in the closing sentence of this paragraph.